index.css

```diff
@@ -0,0 +1,54 @@
+/* ------------------------|| IMPORTS -----------------------*/
+@import url('./utilities/main.css');
+
+/*--------------------- || HOMEPAGE ----------------------------*/
+
+/* --------------------------- || CATEGORY SECTION ----------------- */
+.main {
+    padding: 2rem 7rem;
+    background-color: #F7F7F7;
```

Comment:
you can use color variables instead of hardcoded colors

Reply:
i will create color variables at last.